profile_inference.py

```
@@ -695,6 +713,605 @@ def run_benchmark_mode(dit_handler, llm_handler, args, timer: PreciseTimer):
+        if passing:
+            min_pass = passing[0]
+            print(f"  {capability_name}:")
+            print(f"    Minimum tier:  {min_pass['tier']} ({min_pass['tier_gb']}GB)")
+            print(f"    Peak VRAM:     {min_pass.get('peak_vram_gb', 0):.2f}GB")
+            if failing:
+                max_fail = failing[-1]
+                print(f"    Last failure:  {max_fail['tier']} ({max_fail['tier_gb']}GB) — {max_fail.get('error', 'unknown')[:60]}")
+        else:
+            if failing:
+                print(f"  {capability_name}:")
+                print(f"    ❌ No tier passed this test. All tested tiers failed.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)